### PR TITLE
Fix Code Scanning warnings

### PR DESCRIPTION
**Pull request by aeisenberg**
Removes unnecessary code.

### CODE REVIEW

File: extensions/ql-vscode/src/test-adapter.ts [hunk 1]
```diff
@@ -220,14 +220,13 @@ export class QLTestAdapter extends DisposableObject implements TestAdapter {
...
-        ? 'errored'
-        : 'failed';
+          ? 'errored'
+          : 'failed';
       let message: string | undefined;
       if (event.diff?.length) {
         message = ['', `${state}: ${event.test}`, ...event.diff, ''].join('\n');
         testLogger.log(message);
       }
-      (event.diff || []).join('\n');
```
Comment (adityasharad) [inline extensions/ql-vscode/src/test-adapter.ts:230]: Did we mean to use this above instead of `...event.diff`?

**Review by adityasharad — APPROVED** [review]

File: extensions/ql-vscode/src/test-adapter.ts (hunk 1 above)
Comment (aeisenberg) [reply extensions/ql-vscode/src/test-adapter.ts:230]: No...in the previous line, it is testing that `event.diff` is truthy.

Comment (adityasharad) [reply extensions/ql-vscode/src/test-adapter.ts:230]: And it's putting it into `message`

**Comment by aeisenberg** [comment]
Hmmm...flaky test. I should track down why.

**Pull request merged**